pkg/drivers/kic/oci/cli_runner.go

```diff
@@ -148,8 +148,14 @@ func runCmd(cmd *exec.Cmd, warnSlow ...bool) (*RunResult, error) {
 				out.WarningT(`Executing "{{.command}}" took an unusually long time: {{.duration}}`, out.V{"command": rr.Command(), "duration": elapsed})
 				// Don't show any restarting hint, when running podman locally (on linux, with sudo). Only when having a service.
 				if cmd.Args[0] != "sudo" {
```

Comment:
The logic to determine if this is Docker should be improved so as not to require the clarifying comment. It's pretty obscure.

Reply:
This is a compromise that @afbjorklund suggested to avoid code complexity and since we do not allow running docker with “Sudo” , it is also most efficient way

Reply:
If the runner can't be made aware of which environment it is in, perhaps it's a good sign that it should be passing the signal upwards rather than providing advice for an environment. For instance, returning duration upwards in RunResult.

If that isn't feasible, lets simplify this check, and make it more accurate at the same time. Is there a way to check if the OCI is Docker (based on either the object, cmd.Args, or some other magic)? Ideally, this should say something like:

```
if isDocker() {
  .. show docker advice ...
}
```

For instance, this function could simply check if cmd arg 0 or 1 is `docker`.